src/partials/10_header.html

```diff
@@ -57,8 +57,11 @@
   </div>
   <div class="menu-bar">
     <div class="container">
-      <div class="row align-items-center">
-        <div class="col">
+      <div
+        class="row align-items-center"
+        style="display: flex;justify-content: center;"
+      >
+        <div class="col" style="display: flex;justify-content: center;">
```

Comment:
Czy jest jakiś powód , dlaczego style są osadzone bezpośrednio w atrybucie `style`? Raczej należy unikać takiego zapisu, nie jest to dobra praktyka

Reply:
poprawione - proszę o merge